pkg/kubeadm/master/manifests.go

```diff
@@ -128,6 +160,24 @@ func etcdVolumeMount() api.VolumeMount {
 	}
 }
 
+// certsVolume exposes host SSL certificates to pod containers.
+func certsVolume(s *kubeadmapi.KubeadmConfig) api.Volume {
+	return api.Volume{
+		Name: "certs",
+		VolumeSource: api.VolumeSource{
+			// TODO make path configurable
+			HostPath: &api.HostPathVolumeSource{Path: "/etc/ssl/certs"},
+		},
+	}
+}
+
+func certsVolumeMount() api.VolumeMount {
```

Comment:
So, actually this also exposes host certificates to container. Why? Because most of the times you won't be able to validate certificates because you're missing root certificates, etc.

I agree that the host path should be configurable, and IIRC there's a TODO in the code.